packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -57,6 +70,7 @@ async function fundWallet(
     faucetHost?: string
     faucetPath?: string
     amount?: string
+    memo?: memo
```

Comment:
Why do we want people to be able to specify their own memos? Isn't this just for tracking faucet use?

Reply:
The format of the custom memos used by dev advocates, the learning portal, etc. would be another good thing to define in an XLS spec.